src/Native/Runtime/coreclr/gcinfotypes.h

```diff
@@ -543,8 +542,8 @@ inline void GetInfoHdr(int index, InfoHdr * header)
 
 PTR_CBYTE FASTCALL decodeHeader(PTR_CBYTE table, UINT32 version, InfoHdr* header);
 
-BYTE FASTCALL encodeHeaderFirst(const InfoHdr& header, InfoHdr* state, int* more, int *pCached);
-BYTE FASTCALL encodeHeaderNext(const InfoHdr& header, InfoHdr* state, BYTE &codeSet);
+Int8 FASTCALL encodeHeaderFirst(const InfoHdr& header, InfoHdr* state, int* more, int *pCached);
```

Comment:
Could you please use standard C types for this: `int8_t`, etc.

Reply:
Fixed